src/Test.UnitTests.Sarif/Visitors/SuppressVisitorTests.cs

```diff
@@ -16,98 +17,109 @@ public class SuppressVisitorTests
     {
         private const int DateTimeAssertPrecision = 500;
 
-        [Fact]
-        public void SuppressVisitor_ShouldFlowPropertiesCorrectly()
+        [Theory]
+        [InlineData("", "some suppress justification", false, false, 0, SuppressionStatus.Accepted, null)]
```

Comment:
>[InlineData( [](http://example.com/codeflow?start=8&length=12)

we have seen the Theory InlineData tests using reflection historically caused performance issue , the tests discovering and execution performance is much worse compare to Fact tests.

it can be easily converted to a Fact test by iterating the list of test data using a for loop in the function.

Reply:
@yongyan-gh Understood, I can revert this. The reason I changed it was because it was difficult to tell which test in the test data was failing. Would you be opposed to breaking the for loop in a single test case out into multiple tests?

Reply:
that's definitely works, thank you!

Reply:
@yongyan-gh Test cases updated. Please review those updates.

Reply:
:thumbsup:

#1660